.pre-commit-pr-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+-   repo: https://github.com/Flux7Labs/central-repo-git-hooks.git
+    rev: feat/US3719
```

Comment:
Should it be feature here ? or master ?

Reply:
We kept that branch during testing. Updated to master. @KritikaA06